### PR TITLE
OCPBUG3826: Added a note under cluster-wide proxy installation related to wait-for command.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.10+

Issue:
[OCPBUG-3826](https://issues.redhat.com/browse/OCPBUGS-3826)
Added a note when the installer times out under cluster-wide proxy installation.

Link to docs preview:
[Preview 1](https://58319--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-configure-proxy_installing-openstack-installer-custom)
[Preview 2](https://58319--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-deploying-bare-metal-machines_installing-openstack-installer-custom)

QE review:
- [ ] QE has approved this change.

Additional information:
[Proxy](https://github.com/openshift/installer/blob/master/docs/user/customization.md#proxy)
[Deploying a cluster with bare metal machines](https://github.com/openshift/installer/blob/master/docs/user/openstack/deploy_baremetal_workers.md#considerations-when-deploying-bare-metal-workers)

### CODE REVIEW

File: modules/installation-configure-proxy.adoc [hunk 1]
```diff
@@ -237,6 +237,16 @@ bundle.
 The installation program does not support the proxy `readinessEndpoints` field.
 ====
 
+[NOTE]
+====
+If the installer times out, restart and then complete the deployment by using the wait-for command of the installer. For example:
```
Comment (JoeAldinger) [inline modules/installation-configure-proxy.adoc:242]:
```suggestion
If the installer times out, restart and then complete the deployment by using the `wait-for` command of the installer. For example:
```

File: modules/installation-configure-proxy.adoc [hunk 2]
```diff
@@ -237,6 +237,16 @@ bundle.
...
+
+[source,yaml]
```
Comment (JoeAldinger) [inline modules/installation-configure-proxy.adoc:244]: I _think_ this is a terminal command so it should be [source,terminal]. I'm not sure though and would advise checking with SME. If it is a terminal command I'd also add the $.

File: modules/installation-configure-proxy.adoc [hunk 3]
```diff
@@ -237,6 +237,16 @@ bundle.
...
+----
+./openshift-install wait-for install-complete --log-level debug
```
Comment (JoeAldinger) [inline modules/installation-configure-proxy.adoc:246]:
```suggestion
$ ./openshift-install wait-for install-complete --log-level debug
```

**Review by JoeAldinger — COMMENTED** [review]
A question and a few suggestions.

File: modules/installation-configure-proxy.adoc [hunk 4]
```diff
@@ -237,6 +237,16 @@ bundle.
 The installation program does not support the proxy `readinessEndpoints` field.
 ====
 
+[NOTE]
```
Comment (JoeAldinger) [inline modules/installation-configure-proxy.adoc:240]: In the preview this note doesn't look indented properly. You might need a + on line 239

File: modules/installation-configure-proxy.adoc (hunk 2 above)
Comment (subhtk) [reply modules/installation-configure-proxy.adoc:244]: I have checked with SME. He confirmed it must be a [source,terminal]. I would make the changes.

File: modules/installation-configure-proxy.adoc (hunk 3 above)
Comment (subhtk) [reply modules/installation-configure-proxy.adoc:246]: Ack.

File: modules/installation-configure-proxy.adoc (hunk 4 above)
Comment (subhtk) [reply modules/installation-configure-proxy.adoc:240]: Ack.

File: modules/installation-configure-proxy.adoc (hunk 1 above)
Comment (subhtk) [reply modules/installation-configure-proxy.adoc:242]: Ack.

**Comment by stevsmit** [comment]
lgtm

**Pull request merged**